client/src/test/java/tds/session/SessionTest.java

```diff
@@ -19,4 +22,64 @@ public void shouldBeInstantiated() {
         assertThat(session.getType()).isEqualTo(0);
         assertThat(session.getStatus()).isEqualTo("closed");
     }
+
+    @Test
+    public void shouldBeOpen() {
```

Comment:
I think we have a mix of naming conventions right now for tests.  I remember seeing "itShould..." in other places like here: https://github.com/SmarterApp/TDS_ExamService/blob/develop/service/src/test/java/tds/exam/services/impl/ExamQueryServiceTest.java#L31

It honestly doesn't matter to me which we use, but let's decide and be consistent.